src/fastga/models/performances/mission/dynamic_equilibrium.py

```diff
@@ -392,86 +403,56 @@ def equation_outer(
 
         return np.array([f1, f2])
 
+    def add_flight_point(self, flight_point: FlightPoint = None, equilibrium_result: tuple = None):
 
-def save_df(
-    time,
-    altitude,
-    distance,
-    mass,
-    v_tas,
-    v_cas,
-    rho,
-    gamma,
-    equilibrium_result,
-    thrust_rate,
-    sfc,
-    name: str,
-    existing_dataframe: pd.DataFrame = None,
-):
-    """
-    Method to save mission point to a pandas dataframe file for further post-processing
-
-    :param time: mission time in seconds
-    :param altitude: flight altitude in meters
-    :param distance: flight distance in meters
-    :param mass: aircraft current mass
-    :param v_tas: true air speed in m/s
-    :param v_cas: calibrated air speed in m/s
-    :param rho: air density in kg/m3
-    :param gamma: slope angle in degree
-    :param equilibrium_result: result vector of dynamic equilibrium
-    :param thrust_rate: thrust rate at flight point
-    :param sfc: sfc at flight point
-    :param name: phase name
-    :param existing_dataframe: Dataframe
-    """
+        """
+        Method to add single flight_point to a list of flight_point and treats equilibirum_result at the same time
+
+        :param equilibrium_result: result vector of dynamic equilibrium
+        :param flight_point: the flight_point to add
+        """
+
+        if flight_point is not None:
+            if equilibrium_result is not None:
+                flight_point.alpha = float(equilibrium_result[0]) * 180.0 / math.pi
```

Comment:
```suggestion
                flight_point.alpha = float(equilibrium_result[0]) * 180.0 / np.pi
```
Related to issue #186